src/components/Notifications/types.ts

```diff
@@ -18,13 +21,25 @@ type RawDecentralandNotification<T extends DecentralandNotificationType, M> = {
   metadata: M
 }
 
-export type DecentralandNotificationType =
-  | 'item_sold'
-  | 'royalties_earned'
-  | 'bid_accepted'
-  | 'bid_received'
+export const enum DecentralandNotificationType {
```

Comment:
could u find a way to have this in an array as we had before? There was an array named `CURRENT_AVAILABLE_NOTIFICATIONS` which had all the notifications that the frontend supports or can render. This is needed because there can be generated notifications on the backend that the frontend doesn't know how to render so we need to filter them out on the frontend side. `CURRENT_AVAILABLE_NOTIFICATIONS` will be used to do that